frontend/src/Summary.js

```diff
@@ -1,10 +1,39 @@
+/* eslint-disable import/no-duplicates */
+// eslint-disable-next-line no-unused-vars
 import { useTasksCount } from "./api/api.hooks";
+import { useTasksQuery } from "./api/api.hooks";
 
 export const Summary = () => {
-  const { data: tasksCount } = useTasksCount();
+  const { data: tasks } = useTasksQuery();
+  var taskEffort = 0;
```

Comment:
Please use `let` rather than `var` to make a better use of the ES6 features. Variables declared by `var` keyword are scoped to the immediate function body (hence the function scope) while `let` variables are scoped to the immediate enclosing block denoted by { } (hence the block scope).